fails when running `npm test`

Hi

I'm using nyc with mocha and coveralls
I've setup my .travis.yml and package.json as specified on the readme

The test is passing on travis and test coverage are send to coveralls.
But when I do `npm test` on my computer, it fails

```
> annotation-parser@2.5.2 test C:\perso\node-annotation-parser
> nyc mocha

events.js:85
      throw er; // Unhandled 'error' event
            ^
Error: spawn mocha ENOENT
    at exports._errnoException (util.js:746:11)
    at Process.ChildProcess._handle.onexit (child_process.js:1053:32)
    at child_process.js:1144:20
    at process._tickCallback (node.js:355:11)
    at Function.Module.runMain (module.js:503:11)
    at startup (node.js:129:16)
    at node.js:814:3
npm ERR! Test failed.  See above for more details.
```

The repo: https://github.com/mastilver/node-annotation-parser

I've set `test` by `./node_modules/.bin/nyc.cmd ./node_modules/.bin/mocha.cmd`

It's now outputting:

```
> sails-hook-annotation-router@0.1.1 test C:\perso\sails-hook-annotation-router
> nyc.cmd ./node_modules/.bin/mocha.cmd

'.' is not recognized as an internal or external command,
operable program or batch file.
npm ERR! Test failed.  See above for more details.
```

I've tested the same on linux, it does work.
Unfortunately I need to stick with windows for my job. :/

--- a/bin/nyc.js
+++ b/bin/nyc.js
@@ -81,8 +81,9 @@ if (process.env.NYC_CWD) {
     report(argv)
   } else if (~argv._.indexOf('check-coverage')) {
     foreground(
-      require.resolve('istanbul/lib/cli'),
+      process.execPath,
       [
+        require.resolve('istanbul/lib/cli'),
         'check-coverage',
         '--lines=' + argv.lines,
         '--functions=' + argv.functions,

--- a/index.js
+++ b/index.js
@@ -145,8 +145,17 @@ NYC.prototype.tmpDirectory = function () {
 
 NYC.prototype.mungeArgs = function (yargv) {
   var argv = process.argv.slice(1)
+  argv = argv.slice(argv.indexOf(yargv._[0]))
+  if (!/^(node|iojs)$/.test(argv[0]) &&
+      process.platform === 'win32' &&
+      (/\.js$/.test(argv[0]) ||
+        (!/\.(cmd|exe)$/.test(argv[0]) &&
+        !fs.existsSync(argv[0] + '.cmd') &&
+        !fs.existsSync(argv[0] + '.exe')))) {
+    argv.unshift(process.execPath)
+  }
 
-  return argv.slice(argv.indexOf(yargv._[0]))
+  return argv
 }
 
 module.exports = NYC

--- a/package.json
+++ b/package.json
@@ -13,7 +13,7 @@
   "config": {
     "nyc": {
       "exclude": [
-        "node_modules/"
+        "node_modules"
       ]
     }
   },

--- a/test/nyc-test.js
+++ b/test/nyc-test.js
@@ -33,7 +33,7 @@ describe('nyc', function () {
 
       var nyc = new NYC()
 
-      nyc.cwd.should.match(/nyc\/test\/fixtures/)
+      nyc.cwd.should.equal(path.resolve(__dirname, './fixtures'))
       afterEach()
     })
   })
@@ -241,7 +241,7 @@ describe('nyc', function () {
       })
       nyc.wrap()
 
-      istanbul.config.loadFile.calledWithMatch('test/fixtures/.istanbul.yml').should.equal(true)
+      istanbul.config.loadFile.calledWithMatch(path.join('test', 'fixtures', '.istanbul.yml')).should.equal(true)
       istanbul.Instrumenter.calledWith({
         coverageVariable: '__coverage__',
         embedSource: false,

